openassessment/templates/openassessmentblock/grade/oa_grade_waiting.html

```diff
@@ -16,13 +16,7 @@ <h2 class="step__title">
         <div class="wrapper--step__content">
             <div class="step__content">
                 <div class="grade__value__description">
-                    {% if waiting == 'peer' %}
-                        <p>{% trans "Your response is still undergoing peer assessment. After your peers have assessed your response, you'll see their comments and receive your final grade." %}</p>
-                    {% elif waiting == 'example-based' %}
-                        <p>{% trans "Your response is still undergoing example-based assessment. After your response has been assessed, you'll see the comments and receive your final grade." %}</p>
-                    {% elif waiting == 'all' %}
-                        <p>{% trans "Your response is still undergoing peer assessment and example-based assessment. After your peers have assessed your response and example-based assessment is complete, you'll see your peers' comments and receive your final grade." %}</p>
-                    {% endif %}
+                    <p>{% trans "Your response still needs to undergo some assessment steps. After all assessment steps are complete, you will see comments from peers or course staff and receive your final grade." %}</p>
```

Comment:
@cahrens good point. I'd suggest a minor tweak to your first sentence; the rest sounds good.
"Not all assessment steps have been completed for your response."
@explorerleslie your thoughts?